inst/qml/auditClassicalEvaluation.qml

```diff
@@ -19,456 +19,676 @@
 // When making changes to this file always mention @koenderks as a
 // reviewer in the Pull Request
 
-import QtQuick			2.8
-import QtQuick.Layouts	1.3
-import JASP.Controls	1.0
-import JASP.Widgets		1.0
+import QtQuick									2.8
+import QtQuick.Layouts							1.3
+import JASP.Controls							1.0
+import JASP.Widgets								1.0
 
 Form
 {
 	
 	columns: 			1
 
 	// Extra options
-	CheckBox { name: "workflow"; checked: false; visible: false}
+	CheckBox 
```

Comment:
This extra options section seems familiar, can you make it a custom qml component?